docs/guides/prefect-deploy.md

```diff
@@ -464,6 +464,13 @@ Use the tabs below to explore these two deployment creation options.
 
     Any deployment configuration can be overridden via options available on the `prefect deploy` CLI command when creating a deployment.
 
+    !!! tip "New `prefect.yaml` file flexibility"
+        Traditionally, this file had to be in the root of your repository or project directory and named `prefect.yaml`.
+        Now this file can be located elsewhere and be named differently. To use a custom prefect.yaml file, run the new 
+        form of the deploy CLI command from the root of your project directory: 
+        
+        `prefect deploy --prefect-file path/to/my_file.yaml`
+
```

Comment:
```suggestion
    !!! tip "New `prefect.yaml` file flexibility"
        Traditionally, this file had to be in the root of your repository or project directory and named `prefect.yaml`.
        Now this file can be located elsewhere and named differently. To use a custom deployment configuration file, run the new 
        form of the deploy CLI command from the root of your project directory: 
        
        `prefect deploy --prefect-file path/to/my_file.yaml`

```

is saying "run the new form of the deploy CLI command" necessary? it seems like you'd just pass in the `--prefect-file` option?

Reply:
I think that "new form" is a grandiose way of saying "just pass in a new flag" :) Let me update that to be a little less over the top!